apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -131,16 +145,20 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     event ChangeOverruleWindow(uint64 overruleWindow);
     event ChangeQuietEndingConfiguration(uint64 quietEndingPeriod, uint64 quietEndingExtension);
 
+    // Note: may want to also emit the evm script
     event StartVote(uint256 indexed voteId, address indexed creator, bytes context);
     event PauseVote(uint256 indexed voteId, uint256 indexed challengeId);
     event ResumeVote(uint256 indexed voteId);
     event CancelVote(uint256 indexed voteId);
     event ExecuteVote(uint256 indexed voteId);
+    // Note: maybe we should stick with Vote* or *Vote? E.g. ExtendVote()?
     event VoteQuietEndingExtension(uint256 indexed voteId, bool passing);
 
+    // Note: should we remove stake?
     event CastVote(uint256 indexed voteId, address indexed voter, bool supports, uint256 stake);
```

Comment:
Thought the `stake` is pretty useless here; we already have everything we need (voter and snapshot block) to get it from a call.

Reply:
Definitely, addressed in c93188d423b91af905e0728c956021e16dcf0255